### ISSUE TITLE
ahmedabadmirror.com

### ISSUE THREAD

**Issue by ashutoshgoyal** [X01X012013/AdBlockProtector#17]
http://www.ahmedabadmirror.com/others/specials/Hooked-and-booked-on-health/articleshow/53968999.cms

**Comment by X01X012013** [comment]
Hi, this is fixed into 1.1.1. 

-----
The kernel was updated, adding support for filtering window.document.addEventListener
```JavaScript
activateSetTimeoutFilter(/function \(\)\{if\(\!\_0x/);
activateDocumentAddEventListenerFilter(/function \(\_0x/);
```

**Comment by ashutoshgoyal** [comment]
working fine. thanks.
